rogue-thi-app/lib/hooks/dashboard.js

```diff
@@ -139,7 +139,7 @@ export function useDashboard () {
     const entries = shownDashboardEntries.slice(0)
     const hiddenEntries = hiddenDashboardEntries.slice(0)
 
-    const index = entries.findIndex(x => x.key === key)
+    const index = entries.filter(entry => entry.removable !== false).findIndex(x => x.key === key)
```

Comment:
this looks like a possible bug source where the index returned by `findIndex` does not correspond to the index inside `entries` because of non-removable elements being removed in the immediate list.

Reply:
fixed by @alexhorn in 609b038652478c6dc5ebd6a8771b68893370cc0b